doc-content/enterprise-only/cases/case-management-roles-con.adoc

```diff
@@ -20,7 +20,7 @@ The following example illustrates how the preceding case management concepts app
 
 The `Guest` role assignment affects the specific work of the associated case and is unique to all case instances. The number of users or groups that can be assigned to a role is limited by the `Case Cardinality`, which is set during role creation in the process designer and case definition. For example, the hotel reservation case has only one guest while the *IT_Orders* sample project has two suppliers of IT hardware.
 
-When roles are defined, case management must ensure that roles are not hard coded to a single set of people or groups as part of case definition and that they can differ for each case instance. This is why case role assignments are important.
+When roles are defined, ensure that roles are not hard-coded to a single set of people or groups as part of case definition and that they can differ for each case instance. This is why case role assignments are important.
```

Comment:
Here the hard-coded is used with `-` but in the Assigning a task to a role chapter, the verb is used as hard code. Just noticed it, not sure if it is an issue.

Reply:
Corrected based on doc review input.